packages/editor/src/client/login.tsx

```diff
@@ -147,7 +151,7 @@ export function Login() {
                   flexDirection="column"
                   marginTop={Spacing.Small}
                   alignItems="center">
-                  <p>Or Login with:</p>
+                  <p>{t('Or login with')}:</p>
```

Comment:
@0815pascal why did you leave the `:`?

Reply:
I was thinking that the `:` actually is not something that needs translation; it could also happen that in the process of translation someone deletes it by accident, so I decided to go with that variant.